src/EnergyPlus/SetPointManager.hh

```diff
@@ -1266,6 +1266,10 @@ namespace SetPointManager {
 								 int const CompOpType,
 								 int const ControlNodeNum
 								 );
+
+	Real64
+	CalcSetPoint(Real64 OutLowTemp, Real64 OutHighTemp, Real64 OutDryBulbTemp, Real64 SetTempAtOutLow, Real64 SetTempAtOutHigh);
```

Comment:
@rglazer2 This function is specific to the OA setpoint manager, so it belongs inside the struct DefineOutsideAirSetPointManager   See DefineIdealCondEntSetPointManager::setupSetPointAndFlags for example.  Also, preferred style is to stack arguments in the declaration.